Analysis/Core/include/AnalysisCore/RecoDecay.h

```diff
@@ -182,24 +182,43 @@ class RecoDecay
     return std::atanh(mom[2] / E(mom, mass));
   }
 
-  /// Calculates azimuthal angle from x and y momentum components.
-  /// \param px,py  {x, y} momentum components
-  /// \return azimuthal angle
-  static double Phi(double px, double py)
+  /// Calculates azimuth from x and y components.
+  /// \param x,y  {x, y} components
+  /// \return azimuth within [0, 2π]
+  template <typename T, typename U>
+  static double Phi(T x, U y)
   {
-    // phi = pi+TMath::Atan2(-py,-px)
-    return (double)(o2::constants::math::PI + std::atan2(-py, -px));
+    // conversion from [-π, +π] returned by atan2 to [0, 2π]
```

Comment:
Please, no special characters....

Reply:
You mean the "π"? It should be a Unicode character. Is it not?

Reply:
Exactly...

Reply:
Use of Unicode characters in comments is explicitly encouraged by the [Coding Guidelines](https://rawgit.com/AliceO2Group/CodingGuidelines/master/comments_guidelines.html?showone=Special_Characters#Special_Characters).
What kind of problem do you see with Unicode?